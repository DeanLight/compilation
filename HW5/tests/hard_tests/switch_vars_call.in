

void main()
{
	int y=2;
	switch(y) 
	{
		case 1: 
		int x1 = 1;
		printi(x1);
		case 2: 
		int x2 = 2;
		print("a\n");
		printi(x2);
	};
}
