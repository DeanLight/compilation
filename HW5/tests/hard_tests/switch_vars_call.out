Loaded: ./exceptions.s
a
2